### PR TITLE
fix: generate ssh keys automatically when creating a aks cluster

### PR DESCRIPTION
**Pull request by mojtabaimani**
adding the flag to generate ssh key automatically to the az cli.

### CODE REVIEW

File: tpl/azure-create.sh [hunk 1]
```diff
@@ -138,7 +138,7 @@ if ye kms; then
 fi
 
 echo "echo 'Creating AKS cluster \"$cluster_name\". This will take around 10 minutes...'"
-echo "az aks create -n '$cluster_name' -g '$aks_rg' --zones $(y _ zones) --load-balancer-outbound-ips \$egress_ip_id,\$ingress_ip_id $(y aks.create) $(y aks.nodePoolDefaults)\
+echo "az aks create -n '$cluster_name' -g '$aks_rg' --generate-ssh-keys --zones $(y _ zones) --load-balancer-outbound-ips \$egress_ip_id,\$ingress_ip_id $(y aks.create) $(y aks.nodePoolDefaults)\
```
Comment (Morriz) [inline tpl/azure-create.sh:141]: why make this mandatory when it is now optional?

**Review by Morriz — COMMENTED** [review]
I am not sure if you understand yet that you can already pass this option via the configuration.

File: tpl/azure-create.sh (hunk 1 above)
Comment (j-zimnowoda) [reply tpl/azure-create.sh:141]: @mojtabaimani please follow Maurice advice.
Take a look at `otomi-clusters/clouds/azure/default.yaml:15`
Then in `mojtaba.yaml` file you can set this flag
I just do not understand why some azure users need to set and the others not....

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]

**Comment by mojtabaimani** [comment]
we have `generate-ssh-keys: true` in otomi-clusters repository.

**Pull request closed without merging**